cryptlib.vcxproj

```diff
@@ -160,6 +161,38 @@
       <TargetMachine>MachineX64</TargetMachine>
     </Lib>
   </ItemDefinitionGroup>
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
+    <Lib>
+      <OutputFile>$(OutDir)$(TargetName)d$(TargetExt)</OutputFile>
+    </Lib>
+    <ClCompile>
+      <ProgramDataBaseFileName>$(Platform)\Output\$(Configuration)\$(ProjectName)d.pdb</ProgramDataBaseFileName>

```

Comment:
I suggest to use `$(OutDir)` here and for other such items like it's used for ` <OutputFile>...`:
```
<ProgramDataBaseFileName>$(OutDir)$(TargetName)d.pdb</ProgramDataBaseFileName>
```

Reply:
Thanks! Your suggestion is very well. I applied it to all such items.